src/Files.App/UserControls/Widgets/DrivesWidget.xaml

```diff
@@ -248,5 +229,37 @@
 				</DataTemplate>
 			</ItemsRepeater.ItemTemplate>
 		</ItemsRepeater>
+
+		<!--  Map Drive button  -->
+		<Button
+			Grid.Column="1"
```

Comment:
It seems the column was removed :thinking:

Reply:
Yep :) done